docs/tutorials/on-device-training/ios-app.md

```diff
@@ -0,0 +1,987 @@
+    }
+    
+    var body: some View {
+        VStack {
+            Spacer()
+            
+            ZStack(alignment: .center) {
+                Image(systemName: "mic.fill")
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 100, height: 100)
+                    .foregroundColor( readyToRecord ? .gray: .red)
+                    .transition(.scale)
+                    .animation(.easeInOut, value: 1)
+            }
+            
+            Spacer()
+            
+            Button(action: {
+                readyToRecord = false
+                recordVoice()
+            }) {
+                Text(readyToRecord ? "Record" : "Recording ...")
+                    .font(.title)
+                    .padding()
+                    .background(readyToRecord ? .green : .gray)
+                    .foregroundColor(.white)
+                    .cornerRadius(10)
+                
+            }.disabled(voiceIdentifier == nil || !readyToRecord)
+                .opacity(voiceIdentifier == nil ? 0.5: 1.0)
+            
+            if  inferResult != .notSet {
+                Spacer()
+                ZStack (alignment: .center) {
+                    Image(systemName: inferResult == .user ? "person.crop.circle.fill.badge.checkmark": "person.crop.circle.fill.badge.xmark")
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(width: 100, height: 100)
+                        .foregroundColor(inferResult == .user ? .green : .red)
+                        .animation(.easeInOut, value: 2)
+                    
+                }
+                
+                Text("Probability of User : \(String(format: "%.2f", probUser*100.0))%")
+                    .multilineTextAlignment(.center)
+                    .fontDesign(.monospaced)
+            }
+            
+            Spacer()
+        }
+        .padding()
+        .navigationTitle("Infer")
+        .onAppear {
+            do {
+                voiceIdentifier = try  VoiceIdentifier()
+                
+            } catch {
+                alertMessage = "Error initializing inference session, make sure that training is completed: \(error)"
+                showAlert = true
+            }
+            
+        }
+        .alert(isPresented: $showAlert) {
+            Alert(title: Text("Error"), message: Text(alertMessage), dismissButton: .default(Text("OK")))
+        }
+    }
+}
+
+struct InferView_Previews: PreviewProvider {
+    static var previews: some View {
+        InferView()
+    }
+}
+```
+
+The complete implementation of the `InferView` can be found [here](https://github.com/microsoft/onnxruntime-training-examples/blob/master/on_device_training/mobile/ios/MyVoice/InferView.swift)
+
+### ContentView
+Finally, we will replace the default `ContentView`, so that it will contain buttons to navigate to the `TrainingView` and `InferView`.
```

Comment:
```suggestion
Finally, we will update the default `ContentView` so that it will contain buttons to navigate to the `TrainingView` and `InferView`.
```